src/main/java/error/DukeTaskDoesNotExistException.java

```diff
@@ -0,0 +1,7 @@
+package error;
+
+/**
+ * This exception is thrown when a user tries to mark, unmark, or delete a non-existent Task index number.
```

Comment:
It is good that you have documentation for the custom exceptions.
Pls pay attention to the comment format.

Reply:
Is this one in particular okay? I see that it is in accordance to the rules set in the conventions guide